src/pages/RepoPage/FailedTestsTab/FailedTestsPage/FailedTestsTable/FailedTestsTable.tsx

```diff
@@ -153,26 +216,84 @@ const FailedTestsTable = () => {
     ordering: getSortingOption(sorting),
     filters: {
       branch: branch ? getDecodedBranch(branch) : undefined,
+      flags: flags as string[],
+      // eslint-disable-next-line camelcase
+      test_suites: testSuites as string[],
```

Comment:
Could these values actually be undefined when being passed here, and would that cause any issues when querying? 

Could they be set to empty arrays instead of `undefined` above, and get rid of the type casting?